getting-started.md

```diff
@@ -111,7 +111,7 @@ distribution for more information.
 user@debian:~$
 ```
 
-You can slightly enhance the REPL (the arrow keys do not work,
+You can slightly enhance the SBCL REPL (the arrow keys do not work,
```

Comment:
Not just SBCL repl. CCL, ECL, ABCL work too.

Reply:
right, thanks.